.laminas-ci.json

```diff
@@ -1,12 +1,7 @@
 {
     "exclude": [
-        {"name": "PHPUnit on PHP 5.6 with locked dependencies"},
-        {"name": "PHPUnit on PHP 7.0 with locked dependencies"},
-        {"name": "PHPUnit on PHP 7.1 with locked dependencies"},
-        {"name": "PHPUnit on PHP 7.2 with locked dependencies"},
-        {"name": "PHPUnit on PHP 7.2 with lowest dependencies"},
         {"name": "PHPUnit on PHP 7.3 with lowest dependencies"},
-        {"name": "PHPUnit on PHP 7.3 with locked dependencies"},
-        {"name": "PHPUnit on PHP 7.4 with lowest dependencies"}
+        {"name": "PHPUnit on PHP 7.4 with lowest dependencies"},
+        {"name": "PHPUnit on PHP 8.0 with lowest dependencies"}
```

Comment:
Can we bump the minimum that causes the fail instead? I would like to have excludes removed entirely, if possible

Reply:
i think its phpunit under version 8, i drop support for older version ?

Reply:
@fezfez  Yes, go ahead and drop them.

@weierophinney do you think we should keep otherwise unsupported PhpUnit 8 here? https://phpunit.de/supported-versions.html

Reply:
No.